ui/js/testlibs/mocha/mocha.js

```diff
@@ -0,0 +1 @@
+../../../node_modules/mocha/mocha.js
```

Comment:
I was doing something different earlier and these needed to be in embedded.go.  I think they can be taken out now.